File: java praktikum/prak9/Praktikum 9/src/praktikum/pkg9/Soal3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package praktikum.pkg9;

import java.util.Scanner;

/**
 *
 * @author Nrhys
 */
public class Soal3 {
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner JB = new Scanner(System.in);
        double hb, ongkir, hj, pk, hk, untung;
            System.out.println("inputkan harga beli = ");
            hb = JB.nextDouble();
            System.out.println("inputkan persentase ongkir (hanya angka tidak perlu persennya) = ");
            ongkir = JB.nextDouble();
            System.out.println("inputkan keuntungan = ");
            untung = JB.nextDouble();
            
            ongkir = hb * ongkir / 100;
            hj = hb + untung + ongkir;
            hk = hj - hb;
            pk = hk / hb;
            System.out.println("Persentase Kenaikan adalah "+pk);
    }
}

File: Python praktikum/prak2/soal 3.py
hb = float(input("berapa harga belinya = "))
ongkir = float(input("berapa persentasenya (angka saja) = "))
untung = float(input("Berapa Keuntungan yang diinginkan = "))

ongkir = hb * ongkir / 100
hj = hb + untung + ongkir
hk = hj - hb
pk = hk / hb
print("Persentase Keuntungan = ", pk)
